Update user-guide.rst (#285)
databrickslabs/tempo: docs/user-guide.rst

dataset into a given frequency, then performs interpolation on the sampled time-series dataset.

Possible values for frequency include patterns such as 1 minute, 4 hours, 2 days or simply sec, min, day.

> For the accepted functions to aggregate data, options are 'floor', 'ceil', 'min', 'max', 'mean'. Descriptions of each of these are indicated below: 
> 
> * `floor` - In the time bucket (based on frequency), choose the earliest value by timestamp.
> * `ceil` - In the time bucket, choose the latest value by timestamp.
> * `min` - In the time bucket, choose the minimum (lowest) value regardless of any timestamp. 
> * `max` - In the time bucket, choose the maximum (highest) value regardless of any timestamp. 
> * `mean` - In the time bucket, choose the average value regardless of any timestamp.

Reviewer: ```suggestion
For the accepted functions to aggregate data within time buckets, options are ’floor’, ’ceil’, ’min’, ’max’, ’mean’. Descriptions of each of these are indicated below: 

* `floor` - returns the earliest value by timestamp.
* `ceil` - returns the latest value by timestamp.
* `min` - returns the lowest value regardless of any timestamp. 
* `max` - returns the highest value regardless of any timestamp. 
* `mean` - returns the average value regardless of any timestamp.
```

Reviewer: This is a drive-by comment but it took me a minute to understand `floor` and `ceil` in this context since those are normally used for truncating a digit. Does it make more sense to call these `first` and `last`?

Author: This definitely makes sense. I can add first/last to the frequency names in a subsequent PR; given floor/ceil might be used as hard-coded parameter values right now, we can plan to retire them in 0.2.0 in favor of first/last.
